modules/mockserver/mockserver.go

```diff
@@ -0,0 +1,59 @@
+package mockserver
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// defaultImage is the default MockServer container image
+const defaultImage = "mockserver/mockserver"
```

Comment:
I'd choose a fixed version instead: `mockserver/mockserver:5.15.0`, to make it reliable and not get undesired updates if latest image changed. Please apply it to the examples too 🙏

Reply:
:+1: